makefiles/libfaketime.mk

```diff
@@ -0,0 +1,50 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS         += libfaketime
+LIBFAKETIME_VERSION := 0.9.10
+DEB_LIBFAKETIME_V   ?= $(LIBFAKETIME_VERSION)-1
+
+libfaketime-setup: setup
+	$(call GITHUB_ARCHIVE,wolfcw,libfaketime,$(LIBFAKETIME_VERSION),v$(LIBFAKETIME_VERSION))
+	$(call EXTRACT_TAR,libfaketime-$(LIBFAKETIME_VERSION).tar.gz,libfaketime-$(LIBFAKETIME_VERSION),libfaketime)
+	$(call DO_PATCH,libfaketime,libfaketime,-p1)
+
+ifneq ($(wildcard $(BUILD_WORK)/libfaketime/.build_complete),)
+libfaketime:
+	@echo "Using previously built libfaketime."
+else
+libfaketime: libfaketime-setup
+	sed -i "s/LD_PRELOAD/DYLD_INSERT_LIBRARIES/g" $(BUILD_WORK)/libfaketime/man/faketime.1
```

Comment:
You still need to move this file manually from `build_work` to `build_stage`; alongside that, you need to delete the compressed manpage from `build_stage` that is automatically made...

Reply:
No, make install will install it after it has been modified.

Reply:
I personally don't see this under `build_stage` despite this being true